packages/samples/headless-ssr/cypress/e2e/ssr-search-parameter-manager.cy.ts

```diff
@@ -8,7 +8,17 @@ import {
 const searchStateKey = 'search-state';
 
 describe('headless ssr with search parameter manager example', () => {
-  const route = '/generic';
+  const generic = '/generic';
+  const react = '/react';
+  describe(`when using the ${generic} route`, () => {
```

Comment:
Would be better to use `.each()` on the routes similar to the smoke tests.

Reply:
Yea, I saw the smoke tests were different just after this. Ill adjust !